lib/handlers/put.js

```diff
@@ -70,14 +71,15 @@ async function putStream (req, res, next, stream = req) {
       res.sendStatus(412)
       return next()
     }
+    returnStatus = 204 // if content is sent back change to 200
   } catch (err) {
     resourceExists = false
   }
   try {
     // Fails with Append on existing resource
     if (!req.originalUrl.endsWith('.acl')) await checkPermission(req, resourceExists)
     await ldp.put(req, stream, getContentType(req.headers))
-    res.sendStatus(201)
+    res.sendStatus(returnStatus)
```

Comment:
```suggestion
    res.sendStatus(resourceExists ? 204 : 201)
```
This should be a simplification. Needs to remove `returnStatus` lines 65 and 74
14 current tests to be updated from 201 to 204
Tested the test-suite and the conformance tests. It is OK